src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -17,6 +18,31 @@ public String toString() {
         return columnName;
     }
 
-    public static final String[] COLUMN_NAME_ARRAY =
-            Stream.of(values()).map(AllelicCountTableColumns::toString).toArray(String[]::new);
+    public static final String[] BASIC_COLUMN_NAME_ARRAY = {
```

Comment:
We certainly could but I don't see the benefit: what if toString is not properly overloaded? isn't it better to delegate the mapping to column to the user?